test/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/onflow/cadence-tools/test
 go 1.18
 
 require (
-	github.com/onflow/cadence v0.28.0
+	github.com/onflow/cadence v0.28.1-0.20221017183505-4cb5613cb0fc
```

Comment:
should retarget master now that this commit is merged

Reply:
go doesn't have a way to target the 'master' branch - Can only target a tag or a commit. Given the PR is merged, this now actually points to the master.